glommio/benches/nop.rs

```diff
@@ -0,0 +1,99 @@
+//! Benchmark the noop performance of Glommio. This is a stress test for the executor/reactor
+//! infrastructure.
+
+use glommio::{LocalExecutorBuilder, Task};
+use std::fmt;
+use std::time::{Duration, Instant};
+
+struct Bench {
+    num_tasks: u64,
+    num_events: u64,
+}
+
+const BENCH_RUNS: &'static [Bench] = &[
+    Bench {
+        num_tasks: 100,
+        num_events: 10_000_000,
+    },
+    Bench {
+        num_tasks: 1_000,
+        num_events: 10_000_000,
+    },
+    Bench {
+        num_tasks: 10_000,
+        num_events: 10_000_000,
+    },
+    Bench {
+        num_tasks: 100_000,
+        num_events: 10_000_000,
+    },
+    Bench {
+        num_tasks: 1_000_000,
+        num_events: 10_000_000,
+    },
+];
+
+struct Measurement {
+    total_events: u64,
+    total_tasks: u64,
+    duration: Duration,
+}
+
+impl fmt::Display for Measurement {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            r"completed submission of {} noop events across {} tasks, {} per task
```

Comment:
@gardnervickers A small detail here: I prefer reporting the average cost per individual task -- see the other benchmarks for that.

Reply:
That makes more sense, thanks!